rust/parser/test/mod.rs

```diff
@@ -1189,7 +1203,7 @@ $x owns name @key;
 get $x;"#;
 
     let parsed = parse_query(query).unwrap().into_match();
-    let expected = typeql_match!(var("x").owns(("name", KEY))).get(["x"]);
+    let expected = typeql_match!(var("x").owns(("name", Key))).get(["x"]);
```

Comment:
`KEY` the constant (which used to stand for `IsKey::Yes`) is replaced by `[Annotation::]Key` the enum variant.